source/MaterialXTest/MaterialXFormat/XmlIo.cpp

```diff
@@ -225,32 +225,3 @@ TEST_CASE("Load content", "[xmlio]")
     mx::DocumentPtr nonExistentDoc = mx::createDocument();
     REQUIRE_THROWS_AS(mx::readFromXmlFile(nonExistentDoc, "NonExistent.mtlx", mx::FileSearchPath(), &readOptions), mx::ExceptionFileMissing&);
 }
-
-TEST_CASE("In memory xincludes", "[xmlio]")
```

Comment:
The test itself here is for xinclude logic, and happens to use the Looks.mtlx file.
In this case the test should remain but just reference any other file and test for a child other than `testLooks1.

Reply:
If you have the bandwidth to help refactor this test, without requiring references to Looks.mtlx, that would be much appreciated!